modules/client-configuration/pages/client-config-overview.adoc

```diff
@@ -5,10 +5,8 @@
 
 Registering clients is the first step after installing {productname}, and most of the time you spend with {productname} is spent on maintaining those clients.
 
-{productname} is compatible with a range of client technologies: you can install traditional or Salt clients, running {sle} or another Linux operating system, with a range of hardware options.
+{productname} is compatible with a range of client technologies: you can install clients, running {sle} or another Linux operating system, with a range of hardware options.
```

Comment:
Without specifying "traditional or Salt" the part "you can install clients" does not really fit IMO. What is "a range of client technologies" referring to now? Just the fact that the client distribution can be one of many?

Reply:
Also shortened now.